monetdbe/_cffi/internal.py

```diff
@@ -259,31 +256,49 @@ def append(self, table: str, data: Mapping[str, np.ndarray], schema: str = 'sys'
             raise exceptions.ProgrammingError(error)
 
         work_columns = ffi.new(f'monetdbe_column * [{n_columns}]')
+        work_objs = []
         # cffi_objects assists to keep all in-memory native data structure alive during the execution of this call
-        cffi_objects = []
+        cffi_objects = list()
```

Comment:
i think in an other PR or a previous commit i merged `work_objs` and `cffi_objects`, since afaik they do the same thing, or not?